hack/build.sh

```diff
@@ -109,13 +109,13 @@ if [ -z "$SKIP_BLOG" ]; then
   # For the reasoning behind all this.
   echo 'Converting all links in GitHub source files to Hugo supported relative links...'
   # Convert relative links to support Hugo
-  find . -type f -path '*/content/*.md' ! -name '*_index.md' ! -name '*index.md' ! -name '*README.md' \
+  find . -type f -path '*/content/*.md' ! -name '*index.md' ! -name '*README.md' \
```

Comment:
@julz I'm wondering if this file is redundant now that we use MkDocs?

Reply:
we unfortunately still need this because this bit of the script is building the (still Hugo-based, for now) community and blog sections. @JeffNeff could you drop the changes in this file please?